ResearchKit/Common/ORKResult.h

```diff
@@ -926,6 +926,35 @@ ORK_CLASS_AVAILABLE
 
 @end
 
+/**
+ The `ORKMultipleComponentQuestionResult` class represents the choice
+ answers from a multiple-component picker-style choice-based answer format.
+ 
+ For example, an `ORKMultipleValuePickerAnswerFormat` produces an `ORKChoiceQuestionResult` object.
```

Comment:
should it be ORKMultipleComponentQuestionResult instead of ORKChoiceQuestionResult?